ovos_utils/json_helper.py

```diff
@@ -12,13 +13,16 @@ def merge_dict(base, delta, merge_lists=False, skip_empty=False, no_dupes=True):
             merge_lists: if a list is found merge contents instead of replacing
             skip_empty: if an item in delta is empty, dont overwrite base
             no_dupes: when merging lists deduplicate entries
+            new_only: only merge keys not yet in base
     """
 
     for k, d in delta.items():
         b = base.get(k)
         if isinstance(d, dict) and isinstance(b, dict):
-            merge_dict(b, d)
+            merge_dict(b, d, merge_lists, skip_empty, no_dupes, new_only)
```

Comment:
I think the docstring should make the meaning of "recursive" more obvious, especially now that it includes parameters. It's clear when you read the code, but it might not be obvious right away.

"Recursively merges two dictionaries, including dictionaries within dictionaries." I'm not sure that's any clearer than the way you have it, but it's worth some thought.